testapi.pm

```diff
@@ -1959,8 +1941,8 @@ I<Currently only qemu backend is supported.>
 
 =cut
 
-sub save_storage_drives {
-    my $filename ||= $autotest::current_test->{name};
+sub save_storage_drives ($filename = undef) {
```

Comment:
This seems like a new feature that's not really reflected in the commit message or tests.

Reply:
Hm, I thought it to be effectively equivalent to the previous behaviour. What cases do you see where use cases behave differently?

Reply:
The previous version of the function did not accept an argument and only ever used `$autotest::current_test->{name}` (the `my $filename ||=` was probably a typo).

Reply:
> save_storage_drives([$filename]);

Looks like it was meant to have an argument though, judging by the documentation. More reason to actually test it. :)

Reply:
Yeah sure. The previous version did not accept any argument so what I did is change the function to merely do what the documentation promises but still only as an option.